### DISCUSSION TITLE
PRIDE Archive new API (version 2)

### DISCUSSION THREAD

**Discussion by ypriverol** [PRIDE-Archive/pride-resources#9] [Announcements]
Recently, PRIDE Archive Restful API has been redesign and implemented: [https://www.ebi.ac.uk/pride/ws/archive/v2/swagger-ui.htm](https://www.ebi.ac.uk/pride/ws/archive/v2/swagger-ui.htm). Among different features, the user can now:

- [x] programmatically search for projects
- [x] filter by project features, submission dates, file types, etc. 
- [x] pagination
- [x] full documentation (https://www.ebi.ac.uk/pride/ws/archive/v2/docs/api-guide.html)
- [x] statistics
- [x] identified spectra in PRIDE Spectra Archive. 
  
The previous API (No actively maintained) can be found in the following url: https://www.ebi.ac.uk/pride/ws/archive/

**Comment by weiheng-liao** [comment]
Hi Yasset!  
Recently, we developed a public proteomics dataset search framework, [Patpat](https://github.com/henry-leo/Patpat), based on the PRIDE API. However, I found some problems in Patpat today, and I noticed that `https://www.ebi.ac.uk/pride/ws/ archive/v2/spectra` disappeared. Will it reappear?
Good day!
Weiheng Liao
